### DISCUSSION TITLE
Resources page review

### DISCUSSION THREAD

**Discussion by brianzelip** [Relay-Maryland/relaymaryland.com#15] [General]
@j-p-courneya I edited the Resources content for brevity and grammar. Review it to see what you think.

Also - the URL is needed for the 'Report security concerns' section...I couldn't find that one.

**Comment by j-p-courneya** [comment]
@brianzelip 

I'd remove 3 all together. I think number 2bin the list covers as far as anyone official in the neighborhood would want we don't have a board member in charge of security. I can go modify it as well right?

**Reply by brianzelip** [reply]
@j-p-courneya im not sure what you’re talking about here. Maybe RIA officers? If so here I’m asking about your feedback on the Resources page.

However, yes you can modify whatever. Lemme know if you have any questions.

**Reply by j-p-courneya** [reply]
@brianzelip 

Yeah I realized that I referenced the thing you were inquiring about based on how it appeared in the discussion post I wrote about resources. There the link in question is list item 3. When you revised things you got rid of the numbered items so the location of the item was not clear. In any case in reference to the original question. I'd just dump that as it's extra stuff that truly wasn't relevant to the resources I want to mention on the site
